crates/alkahest/src/maplist.rs

```diff
@@ -177,6 +194,40 @@ impl Map {
         }
     }
 
+    fn fixup_children(
+        &mut self,
+        source: &mut Scene,
+        new_parent: Entity,
+        children: &Children,
+        new_selected: &mut Option<Entity>,
+        selected: &Option<Entity>,
+    ) {
+        let mut new_children = Children(SmallVec::new());
+        for child in children.0.iter() {
+            let old_entity_components = source.take_boxed(*child).unwrap();
```

Comment:
use `let Some(...) = ... else { continue };` here. Children cannot guarantee that the entity ID's it's holding actually exist